components/NewPersonView/UnfinishedSubmissions.spec.tsx

```diff
@@ -71,35 +71,6 @@ describe('UnfinishedSubmissions', () => {
               mockInProgressSubmission,
               mockInProgressSubmission,
               mockInProgressSubmission,
-              mockInProgressSubmission,
-              mockInProgressSubmission,
-            ] as InProgressSubmission[],
-            count: 6,
-          },
-        } as SWRResponse<Paginated<InProgressSubmission>, ErrorAPI>;
-
-        return response;
-      });
-
-    render(<UnfinishedSubmissions personId={1} />);
-
-    expect(screen.getByText('and 2 more'));
-    expect(screen.getAllByRole('listitem').length).toBe(5);
-  });
-
-  it('calculates item left based on response count', () => {
```

Comment:
We don't use the count from array length to determine the total number of unfinished submissions for a user, the count is now part of the paginated response.